tests/cross/src/test/scala/tests/hover/HoverScala3TypeSuite.scala

```diff
@@ -173,4 +173,13 @@ class HoverScala3TypeSuite extends BaseHoverSuite {
        |def apply[T](a: T)(using Int): T
        |""".stripMargin.hover
   )
+
+  check(
+    "toplevel-left",
+    """
+      |def foo = <<L@@eft>>("")
+      |""".stripMargin,
```

Comment:
If you add pipe to the first line then both strings will be aligned :D

Reply:
Fixed!